src/rpc/mining.cpp

```diff
@@ -849,6 +849,7 @@ static UniValue getblocktemplate(const JSONRPCRequest& request)
     result.pushKV("capabilities", aCaps);
 
     UniValue aRules(UniValue::VARR);
+    aRules.push_back("csv");
```

Comment:
17946: hmmm... do we need to add Dash-specific rules for some buried deployments too?

Reply:
Some comment to 17946 says:
> [Comments in IRC](http://www.erisian.com.au/bitcoin-core-dev/log-2020-01-17.html#l-49) provide some context: without the information in rules field "miners have no way to know what rules are being used and might produce invalid blocks"

It seems as bitcoin's miners do not look to VersionBits/Deployment status based on 'pindexPrev', because they are external application that call this rpc `getblocktemplate` and uses this flag to know which version to deploy.

I may be wrong, can't say it yet if it's valuable to add any dash's releases... 
In theory this changes maybe useful after this backport: bitcoin#19401: QA: Use GBT to get block versions correct
 but I have no strong opinion, would add dash releases if it would be useful at some moment, but don't want to do it now because I don't want to have one extra place which is needed to update for each BIP9 fork.